java/odsa-jdbc-atp/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.oracle.dev.jdbc</groupId>
+	<artifactId>odsa-jdbc-atp</artifactId>
+	<version>1.0-SNAPSHOT</version>
+
+	<name>odsa-jdbc-atp</name>
+	<description>A simple JDBC + ODSA ATP app example</description>
+	<url>
+		https://medium.com/oracledevs/odsa-for-java-developers-introduction-to-oracle-database-service-for-azure-part-1-bff24c787055</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>21</maven.compiler.source>
+		<maven.compiler.target>21</maven.compiler.target>
+	</properties>
+
+	<dependencies>
+		<!-- Oracle JDBC JARs -->
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>23.4.0.24.05</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>oraclepki</artifactId>
+			<version>19.11.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.security</groupId>
+			<artifactId>osdt_core</artifactId>
```

Comment:
osdt_* are no longer needed when using 23.4.

Reply:
Ok, thanks for orienting me on that. It's done now, please check commit 9a2b32888bc6ff6a4fa725870638c9ef38b7d777